core/storage/suggestion/gae_models.py

```diff
@@ -72,6 +74,12 @@
 # The delimiter to be used in score category field.
 SCORE_CATEGORY_DELIMITER = '.'
 
+# Threshold time after which suggestion is considered stale and auto-accepted.
+THRESHOLD_TIME_BEFORE_ACCEPT = 14 * 24 * 60 * 60 * 1000
```

Comment:
Are we running this weekly or bi-weekly?

Reply:
I am actually not sure.. Open for your opinion here. It feels weird that I put the threshold as 14 days while the cron job runs weekly. I think weekly is good? (If so i'll make this 7*....) What do you say?

Reply:
Made it weekly. If you feel otherwise, we'll change it.